h5py/h5s.pyx

```diff
@@ -47,7 +47,7 @@ UNLIMITED = H5S_UNLIMITED
 NO_CLASS = H5S_NO_CLASS
 SCALAR   = H5S_SCALAR
 SIMPLE   = H5S_SIMPLE
-_NULL = H5S_NULL
+globals()["NULL"] = H5S_NULL  # "NULL" is reserved in Cython
```

Comment:
Did you intend to drop the leading `_` here?  I don't really understand what is going on here, just aggressively pattern matching.